src/main/kotlin/org/wycliffeassociates/otter/common/data/model/IAudioPlugin.kt

```diff
@@ -0,0 +1,17 @@
+package org.wycliffeassociates.otter.common.data.model
+
+import io.reactivex.Completable
+import java.io.File
+
+interface IAudioPlugin {
+    var id: Int
+    var name: String
```

Comment:
not sure if this should be a merge of the database entity as well as the audio plugin interface functionality

Reply:
I agree. Just not sure how to resolve. I think splitting into a db model entity and the functionality interface would require us to pass around both objects in UserPreferences. We'd need the model (with id) if the user needed to update the plugin (say, to fix the executable path) and we'd need the functional object to launch/delete the plugin.

Reply:
Alternatively, we could go back to having a launch use case and an AudioPluginLauncher interface/implementation to only need the db model entity. Definitely open to other ideas though